.github/workflows/schedule.yaml

```diff
@@ -0,0 +1,10 @@
+---
+    name: 'Close stale issues and PRs'
+
+    'on':
+      schedule:
+        - cron: '30 1 * * *'
+
+    jobs:
+      stale:
+        uses: broadinstitute/shared-workflows/.github/workflows/close-stale.yaml@v2.5.0
```

Comment:
Where are we setting 60days?  Is that a default for the above workflow?  I am a bit fan of explicitly stating a value even if it is default for values we care about.  In this case 60days.

Reply:
yeah, it's defined in the shared workflow https://github.com/broadinstitute/shared-workflows/blob/main/.github/workflows/close-stale.yaml#L37-L40 

I think a parameter would be a good feature add for the next workflow release!

Reply:
ok see that Broad has set our implementation of the close-stale workflow to 60days.  ok I am fine with that.